mbed_targets/_internal/target_attributes.py

```diff
@@ -100,3 +105,19 @@ def _extract_target_attributes(all_targets_data: Dict[str, Any], target_name: st
     accumulated_attributes = get_accumulating_attributes_for_target(all_targets_data, target_name)
     target_attributes.update(accumulated_attributes)
     return target_attributes
+
+
+def _extract_core_labels(build_attributes: Dict[str, Any]) -> Set[str]:
```

Comment:
Since `core_labels` relies only on `core_name` I think it'd be better if that was passed instead. That way the knowledge of how `build_attributes` is structured is moved up the call stack.

Reply:
Yeah that would be much better 👍

Reply:
b0b42e5